doc/Maintainers-Guide.md

```diff
@@ -1,4 +1,4 @@
-# Maintainer's Guide
+# Maintainer’s Guide
```

Comment:
@jasonzio curly quotes are very legit and should be used in writing more. Straight quotes (') and (") actually break the markdown syntax highlighting in vi. This fixes that, too.